tests/testthat/test-pagination.R

```diff
@@ -64,14 +64,13 @@ test_that("pagination works", {
   ##                                min_siblings = 0,
   ##                                verbose = TRUE))
 
-  expect_error(
+  expect_message(
```

Comment:
there is one test here. do you want more? @Melkiades

Reply:
This does not test how it looks for all the cases (e.g. w/ different combinations of content, data and label rows), if the pagination counts are correct and if the exporters work accordingly for these cases. In general, my asking for visual tests, in particular, serves us to review precisely what is the purpose of this modification and to appreciate the extent of changes. For example, I am quite sure that we would need to change a lot of documentation to reflect this change. This needs to be reviewed and added to the PR